pytest.ini

```diff
@@ -8,4 +8,4 @@
 # it under the terms of the GPLv3 license; see LICENSE file for more details.
 
 [pytest]
-addopts = --ignore=docs --cov=cernopendata_client --cov-report xml
+addopts = --ignore=docs --cov=cernopendata_client --cov-report=xml
```

Comment:
What about using both options so that people would see also the line coverage when running locally?

```
addopts = --ignore=docs --cov=cernopendata_client --cov-report=term-missing --cov-report=xml
```

Reply:
Done 👍